<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LootTableEntry</class>
 <widget class="QTabWidget" name="LootTableEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::South</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="general">
   <attribute name="title">
    <string>General</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="typeCmobo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Empty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Item</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Loot table</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tag</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Group</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dynamic</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="empty"/>
      <widget class="QWidget" name="withName">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="TrueFalseBox" name="tagExpandCheck">
          <property name="text">
           <string>Expand</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="withChild">
       <layout class="QFormLayout" name="formLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Select mode</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QRadioButton" name="selectSeqRadio">
             <property name="text">
              <string>Until the first entry fails</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="selectAllRadio">
             <property name="text">
              <string>All entries</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="selectAltRadio">
             <property name="text">
              <string>Only the first successful entry</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="entryGroupLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add and remove entries in the &lt;a href=&quot;/entries&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Entries&lt;/span&gt;&lt;/a&gt; tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="indent">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QSpinBox" name="weightSpin">
      <property name="specialValueText">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="qualityLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Bonus weight from luck.</string>
      </property>
      <property name="text">
       <string>Quality:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QSpinBox" name="qualitySpin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="specialValueText">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="functions">
   <attribute name="title">
    <string>Functions</string>
   </attribute>
  </widget>
  <widget class="QWidget" name="condtions">
   <attribute name="title">
    <string>Conditions</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="addCondButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>301</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QScrollArea" name="conditionsArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="conditionsContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>443</width>
         <height>334</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="entries">
   <attribute name="title">
    <string>Entries</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="addEntryButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QScrollArea" name="entriesArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="entriesContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>443</width>
         <height>334</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>301</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrueFalseBox</class>
   <extends>QCheckBox</extends>
   <header location="global">truefalsebox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
